### ISSUE TITLE
emcc can't find wasm-ld

### ISSUE THREAD

**Issue by DragonOsman** [juj/emsdk#188]
Just now I tried to run em++ after setting `EMCC_WASM_BACKEND` and `EMCC_DEBUG` variables to 1 and I got this error:
```
shared:ERROR: WASM_BACKEND selected but could not find lld (wasm-ld): C:/Users/Osman/emsdk/clang/e1.38.20_64bit\wasm-ld.exe
```
I'd ran these commands before that:
```
emsdk install latest --vs2017 --build=Release
emsdk activate latest --vs2017 --build=Release
emsdk_env
```
I'd really like some help here.  Thanks in advance.

**Comment by Fraer** [comment]
Have same issue on mac os, just made `brew install emscripten` and each time i run emcc i have the message:
```
shared:ERROR: emcc: WASM_BACKEND selected but could not find lld (wasm-ld): /usr/bin/wasm-ld
```

is it possible to install emscripten with all stuff needed for wasm ?

**Comment by kripken** [comment]
@Fraer it's possible the `brew` setup only supports fastcomp for now. We should file a bug for that - if you can please do so and link to it here. (I don't have a MacOS machine so I can't provide any info they might ask me if I file a bug there.)

Separately from brew, the wasm backend should work in the official emsdk builds, see https://v8.dev/blog/emscripten-llvm-wasm#testing

**Comment by sbc100** [comment]
Seems like this is an issue with the homebrew package.   Is this the right tracker for such bugs?   Does anyone know who owns that package?

**Comment by kripken** [comment]
I did some investigating here:

 * The main brew repo discourages people from filing issues for things that aren't clearly bugs with STRs, so I didn't file an issue there.
 * I went through commits trying to find who maintains the emscripten release, but it was hard to tell - there are commits to update to new builds, but they are by people that do a lot of other stuff. I'm not sure if those are just automatic or semi-automatic updates, or actual devs that are familiar with it.
 * I pinged the forum post about this that I linked earlier. No response yet.

This is unfortunate since sometimes people use brew and run into issues, like [here](https://twitter.com/richgel999/status/1175222583597260800). In parallel to trying to contact the proper brew people, maybe we should update our docs on the install page to say that only the emsdk is supported officially?

**Comment by sbc100** [comment]
I agree with updating the docs, at least so say that the brew package only supports fastcomp .

**Comment by kripken** [comment]
Docs have been updated. I think we can remove this from the blocker list given that, as suggested by @sbc100 

I'll still try to contact the proper brew people. A week passed since that form post, I'll try filing an issue.

**Comment by kripken** [comment]
I posted a comment on the [latest brew PR that updates emscripten[(https://github.com/Homebrew/homebrew-core/pull/44158#issuecomment-535106817).
